### PR TITLE
Regenerated from API

### PR DESCRIPTION
**Pull request by sbryzak**
Regenerated as a result of API changes.

Related PR: https://github.com/codeready-toolchain/api/pull/265

### CODE REVIEW

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -18,39 +18,38 @@ objects:
       name: registration-service
       namespace: ${NAMESPACE}
     rules:
-      - apiGroups:
```
Comment (alexeykazakov) [inline deploy/registration-service/registration-service.yaml:21]: I guess it didn't rebase correctly. Please do not change this file.

Comment (sbryzak) [reply deploy/registration-service/registration-service.yaml:21]: hmm, that's weird... will try to regenerate again

Comment (MatousJobanek) [reply deploy/registration-service/registration-service.yaml:21]: You need to pull the latest version of registration-service and then regenerate

**Comment by MatousJobanek** [comment]
@sbryzak Please don't forget about the missing changes from the previous PR: https://github.com/codeready-toolchain/host-operator/pull/536#issuecomment-966111359
It doesn't block this particular change, I just noticed that you haven't added it yet.

File: deploy/registration-service/registration-service.yaml (hunk 1 above)
Comment (sbryzak) [reply deploy/registration-service/registration-service.yaml]: Done, I've also made the other config changes as mentioned in https://github.com/codeready-toolchain/host-operator/pull/536

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**